src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -1071,6 +1072,24 @@ private void UpdateDatabase(TransactionOperationContext context, string type, Bl
             }
         }
 
+        private void UpdateEtagForBackup(DatabaseRecord databaseRecord, string type, long index)
+        {
+            switch (type)
+            {
+                case nameof(PutIndexCommand):
+                case nameof(PutAutoIndexCommand):
+                case nameof(DeleteIndexCommand):
+                case nameof(SetIndexLockCommand):
+                case nameof(SetIndexPriorityCommand):
+                case nameof(SetIndexStateCommand):
+                case nameof(EditRevisionsConfigurationCommand):
+                case nameof(EditExpirationCommand):
```

Comment:
Given that we backup the database record itself, what about all the stuff that mutate it?
updating the backup definition, updating ETL, etc?

Reply:
We have to backup the whole thing